### PR TITLE
CEPHSTORA-427 Update master dependencies

### PR DESCRIPTION
**Pull request by rpc-jenkins-svc**
Automated dependency update pull request, see individual commits for details.

### CODE REVIEW

File: component_requirements.yml [hunk 1]
```diff
@@ -1,8 +1,2 @@
 # Do not modify by hand. This file is automatically generated from the required dependencies and the constraints specified for them.
-dependencies:
-- name: rpc-maas
-  ref: 1.7.6
-  ref_type: tag
-  repo_url: https://github.com/rcbops/rpc-maas
-  sha: 2dc9c16e7f552c52fe596e47fbefdc1472c3485b
-  version: 1.7.6
+dependencies: []
```
[the remark this answers is not in the document]
Comment (git-harry) [reply component_requirements.yml]: @mvollman no, sorry about this, it looks like a bug has been introduced by a recent change. It should have looked like:
```diff
diff --git a/component_requirements.yml b/component_requirements.yml
index 9452384..708a279 100644
--- a/component_requirements.yml
+++ b/component_requirements.yml
@@ -1,8 +1,8 @@
 # Do not modify by hand. This file is automatically generated from the required dependencies and the constraints specified for them.
 dependencies:
 - name: rpc-maas
-  ref: 1.7.6
+  ref: 1.7.7
   ref_type: tag
   repo_url: https://github.com/rcbops/rpc-maas
-  sha: 2dc9c16e7f552c52fe596e47fbefdc1472c3485b
-  version: 1.7.6
+  sha: 037cc33aa66673b0bea6ced749d8511d1c45deb1
+  version: 1.7.7
```
It'll take until tomorrow to get the system updated to automatically propose the correct version however you can update this pull request manually with the above diff if you don't want to wait.

[the remark this answers is not in the document]
Comment (mvollman) [reply component_requirements.yml]: No problem.  We will update it manually.  Thanks.

**Comment by letterj** [comment]
LGTM

**Comment by letterj** [comment]
:shipit:

**Pull request merged**